compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -724,13 +725,25 @@ public void visit(BLangPackage pkgNode) {
                 initFnBody.stmts.add(constInit);
             }
         }
+        List<BLangVariable> desugaredGlobalVarList = new ArrayList<>();
 
         pkgNode.globalVars.forEach(globalVar -> {
-            BLangAssignment assignment = createAssignmentStmt(globalVar);
-            if (assignment.expr != null) {
-                initFnBody.stmts.add(assignment);
+            // This will convert complex variables to simple variable
+            if (globalVar.getKind() == NodeKind.TUPLE_VARIABLE) {
+                //Tuple variable will be desugared into block statement node
```

Comment:
Do we need this comment?

Reply:
It's better for readability right

Reply:
IMO it's kinda obvious when we look at what's done in the next few lines.

Reply:
Removed